pkg/vadmin/restore_points_query_vc.go

```diff
@@ -69,5 +71,10 @@ func (v *VClusterOps) genRestorePointsOptions(s *showrestorepoints.Parms, certs
 	opts.Cert = certs.Cert
 	opts.CaCert = certs.CaCert
 
+	// optional query filter options
+	if s.FilterOptions != nil {
```

Comment:
Will the `FilterOptions` always be set? Makes me wonder if it should even be a pointer.

Reply:
Yes, it will always be set.

Reply:
Changed that field to not be a pointer.